spec/features/step_by_step_navigation_spec.rb

```diff
@@ -20,16 +21,97 @@
       then_I_visit_the_navigation_steps_page_again
       and_I_see_my_selected_preferences
     end
+
+    scenario "User schedules publishing" do
+      given_there_is_a_draft_step_by_step_page
+      and_I_visit_the_scheduling_page
+      and_I_fill_in_the_scheduling_form
+      when_I_submit_the_form
+      then_I_should_see "has been scheduled to publish"
+      and_my_step_by_step_should_have_the_status "Scheduled"
+    end
+
+    scenario "User tries to schedule publishing for date in the past" do
+      given_there_is_a_draft_step_by_step_page
+      and_I_visit_the_scheduling_page
+      and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
+      when_I_submit_the_form
+      then_I_should_see "Scheduled at can't be in the past"
+      and_my_step_by_step_should_have_the_status "Draft"
+    end
+
+    scenario "User tries to schedule publishing for an already live step by step" do
+      given_there_is_a_live_step_by_step_page
+      when_I_visit_the_scheduling_page
+      then_I_should_see "There are no changes to schedule!"
+      and_there_should_be_no_schedule_button
+    end
+
+    scenario "User tries to schedule publishing for an already scheduled step by step" do
+      given_there_is_a_scheduled_step_by_step_page
+      when_I_visit_the_scheduling_page
+      then_I_should_see "Scheduled to be published at"
+      and_there_should_be_no_schedule_button
+    end
   end
 
   def given_there_is_a_step_by_step_page_with_navigation_rules
     @step_by_step_page = create(:step_by_step_page_with_navigation_rules)
   end
 
+  def given_there_is_a_draft_step_by_step_page
+    @step_by_step_page = create(:step_by_step_page_with_steps)
+    expect(@step_by_step_page.status[:name]).to eq 'draft'
+  end
+
+  def given_there_is_a_live_step_by_step_page
```

Comment:
This could be replaced with the existing `given_there_is_a_published_step_by_step_page`

Reply:
Done 👍 had to add a refactor commit to consolidate all these methods into the step_nav_steps.rb.